modules/ListButtons.js

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
+import {View, StyleSheet, Dimensions} from 'react-native';
+import MyButton from './MyButton';
+
+const screenWidth = Dimensions.get('screen').width;
+
+class ListButtons extends Component{
```

Comment:
If you don't need state then don't create class for that component.
You can just create and function - its argument will be props from the parents.

Reply:
This is called - stateless component.